fe/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -914,17 +914,29 @@ private void computeFragmentHosts() throws Exception {
                 // there is no leftmost scan; we assign the same hosts as those of our
                 // leftmost input fragment (so that a partitioned aggregation
                 // fragment runs on the hosts that provide the input data)
-                PlanFragmentId inputFragmentIdx = fragments.get(i).getChild(0).getFragmentId();
+
+                // find a input fragment which has a higher parallelism
+                int inputFragmentIndex = 0;
+                int maxParallelism = 0;
```

Comment:
I think `parallelism` rename  to  instance_number is better.

Reply:
👌